maker/src/routes.rs

```diff
@@ -1,52 +1,70 @@
+use crate::bitmex::Quote;
+use anyhow::Result;
 use http_api_problem::HttpApiProblem;
+use http_api_problem::StatusCode;
 use rocket::serde::json::Json;
 use rocket::serde::Deserialize;
 use rocket::serde::Serialize;
-
-#[derive(Serialize, Deserialize, Debug)]
-pub struct Price(u64);
-
-impl From<u64> for Price {
-    fn from(val: u64) -> Self {
-        Self(val)
-    }
-}
+use rocket::State;
+use rust_decimal::prelude::ToPrimitive;
+use rust_decimal::Decimal;
+use rust_decimal_macros::dec;
+use tokio::sync::watch;
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct Offer {
-    bid: Price,
-    ask: Price,
+    bid: u64,
+    ask: u64,
```

Comment:
I think this should a `Decimal` or `f64`

Reply:
We did already introduce `Decimal` for the amount type, I think we should stick to `Decimal.

Reply:
fixed 3a6fbc18afb3b6de993197f8a35eec8cfc698d08 (maker) and 4c34e9813bfceeb710430ccbf3fd7780978eaa41 (taker)

Decimal can not be used for the taker though as Decimal does not implement the IntoDart trait. We could look into it eventually, but not now.